content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,154 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+content_type: concept
+weight: 10
+---
+
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in {{< glossary_tooltip text="EndpointSlices" term_id="endpoint-slice"
+>}}. This approach tries to keep traffic closer to where it originated from.
+This can help to reduce costs and improve performance.
+
+<!-- body -->
+
+## Motivation
+
+Kubernetes clusters are increasingly deployed in multi-zone environments.
+_Topology Aware Hints_ provides a mechanism to help keep traffic within the zone
+it originated from. This concept is commonly referred to as "Topology Aware
+Routing". The "Hints" here refer to a key implementation detail. The
```

Comment:
I'd strike the `The "Hints" here` sentence and segue with something like:  "When calculating the endpoints for a Service, the EndpointSlice controller considers the topology (region and zone) of each endpoint and populates the `hints` field to allocate it to a zone.  These hints ..."

Reply:
Updated, thanks!